wazero/testdata/wasi_arg.wat

```diff
@@ -0,0 +1,61 @@
+;; $wasi_arg is a WASI command which copies null-terminated args to stdout.
```

Comment:
Copied as is from wasi test, let me know if this makes sense

Reply:
maybe just add a comment that this is a copy of (where)

Reply:
Sure